packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -400,6 +411,33 @@ test.serial('discount bid settled', async t => {
   await assertPayouts(t, seat, currency, collateral, 250n - 231n, 200n);
 });
 
+test.serial('Excessive want in proposal', async t => {
+  const { collateral, currency } = t.context;
+  const driver = await makeAuctionDriver(t);
+
+  await driver.setupCollateralAuction(collateral, collateral.make(1000n));
+  await driver.updatePriceAuthority(
+    makeRatioFromAmounts(currency.make(11n), collateral.make(10n)),
+  );
+
+  const schedules = await driver.getSchedule();
+  t.is(schedules.nextAuctionSchedule?.startTime.absValue, 170n);
+  await driver.advanceTo(170n);
+
+  const seat = await driver.bidForCollateralSeat(
+    currency.make(250n),
+    collateral.make(5000n),
+    makeRatioFromAmounts(currency.make(120n), currency.make(100n)),
+    undefined,
+    collateral.make(5000n),
+  );
+  await t.throwsAsync(() => E(seat).getOfferResult(), {
+    message: 'seat has been exited',
```

Comment:
how do we test the exit reason, 'unable to satisfy want'?

Reply:
https://github.com/Agoric/documentation/pull/791

added to the test